airbyte-webapp/src/locales/en.json

```diff
@@ -628,6 +628,17 @@
   "ui.secretTextArea.hidden": "Contents hidden. Click to show.",
   "ui.stepIndicator.currentStep": "(current step)",
 
+  "airbyte.datatype.WellKnownTypes.json#/definitions/String": "String",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/BinaryData": "Binary Data",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/Number": "Number",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/Integer": "Integer",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/Boolean": "Boolean",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/Date": "Date",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/TimestampWithTimezone": "Timestamp with Timezone",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/TimestampWithoutTimezone": "Timestamp without Timezone",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/TimeWithTimezone": "Time with Timezone",
+  "airbyte.datatype.WellKnownTypes.json#/definitions/TimeWithoutTimezone": "Time without Timezone",
```

Comment:
Ideally, we still want to use the `airbyte.datatype` values below this so that we don't have duplicate strings. The utility function may need a way to translate these types, for example:

`"WellKnownTypes.json#/definitions/TimestampWithTimezone" -> "airbyte.datatype.time_stamp_with_timezone"`

Reply:
I added a map in `useTranslateDataType` utility and updated the strings here.